service/src/main/java/com/ge/predix/acs/service/policy/validation/PolicySetValidatorImpl.java

```diff
@@ -82,11 +91,78 @@ public void removeCachedConditions(final PolicySet policySet) {
 
     @Override
     public void validatePolicySet(final PolicySet policySet) {
+
+        ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
```

Comment:
Default validator factory relies on some sort of bean validator implementation and expression language implementation present in the class path.  Currently, we happen to load hibernate bean validator version 5.3.5.Final (managed by cassandra-all dependency) and jasper apache-el version 8.0.33 (managed by spring-boot-starter-jetty 1.5.3.RELEASE).  

If we decide to go with JSR-303 bean validation annotations to enforce obligations schema and either use bean validator here or @Valid annotation at the controller layer, we have to explicitly specify implementation and version of bean validation and expression language and not rely on other dependencies.  Versions and implementations have to be consistent between ACS model  to use for constraints unit testing and ACS service to use for actual validation.